trend_micro_email_security/assets/logs/trend-micro-email-security.yaml

```diff
@@ -0,0 +1,666 @@
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `request` to `trend_micro_email_security.url`
+          enabled: true
+          sources:
+            - request
+          sourceType: attribute
+          target: trend_micro_email_security.url
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - name: Lookup for `syslog.severity` to `email_severity` field
+          enabled: true
+          source: syslog.severity
+          target: email_severity
+          lookupTable: |-
+            2,Safe
+            4,Suspicious
+            6,Highly suspicious
+            8,Dangerous
+          type: lookup-processor
+        - name: Lookup for `email_severity` to `status` field
+          enabled: true
+          source: email_severity
+          target: status
+          lookupTable: |-
+            Safe,ok
+            Suspicious,warning
+            Highly suspicious,critical
+            Dangerous,emerg
+          type: lookup-processor
+    - type: pipeline
+      name: Remap attributes for audit logs
+      enabled: true
+      filter:
+        query: "service:audit"
+      processors:
+        - type: attribute-remapper
+          name: Map `cs1` to `accountType`
+          enabled: true
+          sources:
+            - cs1
+          sourceType: attribute
+          target: accountType
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `cs2` to `eventType`
+          enabled: true
+          sources:
+            - cs2
+          sourceType: attribute
+          target: eventType
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `cs3` to `affectedDomains`
+          enabled: true
+          sources:
+            - cs3
+          sourceType: attribute
+          target: affectedDomains
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `suser` to `usr.name`
+          enabled: true
+          sources:
+            - suser
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: attribute-remapper
+      name: Map `suser` to `sender`
+      enabled: true
+      sources:
+        - suser
+      sourceType: attribute
+      target: sender
+      targetType: attribute
+      preserveSource: false
+      overrideOnConflict: false
+    - type: status-remapper
+      name: Define `status` as the official status of the log
+      enabled: true
+      sources:
+        - status
+    - type: attribute-remapper
+      name: Map `cs1Label` to `attributeLabels.cs1Label`
```

Comment:
Do you ever think these may be facets in the future? If so, might be worth changing the destination path to have a namespaced suffix

Reply:
@shubhamvekariya-crest - were you able to get a chance to look at this comment? Happy to approve once I know more about this

Reply:
This is now stale due to approach change as discussed with Jason, Closing this PR and we will raise a new one afresh.